src/Assignment2b/Question4.java

```diff
@@ -1,35 +1,24 @@
 package Assignment2b;
 
 import java.util.Scanner;
+import java.util.Stack;
```

Comment:
Implement stack code yourself and use it in future.
Implement all the data structures from base to top in individually.